portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,122 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="navbar.css">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+  
+  <div id = "index">
+    <body>
+      <header class="header">
+        <nav class="navbar">
+          <a href="#index" class="nav-name">Sara Wu</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="#index" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="#about" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#projects" class="nav-link">Projects</a>
+              </li>
+              <li class="nav-item">
+                <a href="#contact" class="nav-link">Contact</a>
+              </li>
+            </ul>
+        </nav>
+      </header>
+
+      <!-- <div class = "content"> -->
+        <!-- ITNRODUCTION -->
+        <section class = "intro">
+          <div class = "padding">
+            <div id="title">
+              Hi, I'm Sara!
+            </div>
+            <div id="subtitle">
+              An aspiring software engineer passionate about 
+              designing and creating for others.
+            </div>
+            <br>
+            <img src="images/linkedin.png" height="40" width="40">
```

Comment:
Same as css, these images refer to local resources (e.g. images/linkedin.png) they should be added to the repository.